vuepress/docs/next/docs/reference/databases.md

```diff
@@ -109,7 +114,7 @@ If the operator does not detect an
 to its default behaviour--creating a matching deployment and
 spinning up a database service from the same namespace. 
 
-If the `spec.dbms` property is not specified on an EntandoApp, the operator will
+If the `spec.dbms` property is not specified on an `EntandoApp`, the operator will
```

Comment:
an EntandoApp is an instance of an EntandoApp CR, and a property is specified *for* an instance, so "specified for"